caffe2/core/blob.h

```diff
@@ -24,10 +27,40 @@ inline bool BlobIsTensorType(const Blob& blob, DeviceType device_type) {
   return tensor && *tensor && tensor->GetDeviceType() == device_type;
 }
 
+inline bool XBlobIsTensorType(const Blob& blob, DeviceType device_type) {
```

Comment:
What does the X mean?

Reply:
nothing, just a temporary name for codemod, we'll change remove the other BlobIsTensorType and change this to BlobIsTensorType after the codemod is done.